data/community_chats.json

```
@@ -46,5 +46,29 @@
         "chat": "xmpp:commteam@muc.xmpp.org?join",
         "web_chat": "https://xmpp.org/chat?commteam",
         "logs": "https://logs.xmpp.org/commteam/"
+    },
+        {
+        "name": "XMPP Summit",
+        "text": "Join here if you plan to attend our summit or get in contact with us at FOSDEM.",
```

```suggestion
        "text": "Room for discussion on and during XMPP Summits",
```

This is from the chat description